app/components/SettingsCell.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { View, Text, StyleSheet, TouchableOpacity, Switch } from 'react-native'
+import { colors } from '../config'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import Divider from './Divider'
+
+const { cellTitle, dateText } = colors
+
+const SettingsCell = ({
+  title,
+  onCellPress,
+  onSwitchValueChange,
+  titleNumLines,
+  iconName,
+}) => {
+  const cellPressed = () => (onCellPress ? onCellPress() : null)
+  const switchValueChanged = (value) =>
+    onSwitchValueChange ? onSwitchValueChange(value) : null
+
+  const { titleText, wrapper, innerContent, icon } = styles
+  return (
+    <TouchableOpacity
+      onPress={cellPressed}
+      activeOpacity={cellPressed == null ? 1 : undefined}
+    >
+      <View style={wrapper}>
+        <Text style={titleText} numberOfLines={titleNumLines}>
+          {title}
+        </Text>
+        <View style={innerContent}>
+          {onCellPress == null ? (
+            <Switch onValueChange={switchValueChanged} />
+          ) : (
+            <Icon style={icon} name={iconName} />
```

Comment:
@MDHayes i think you want me to do it like this.

Reply:
Yeah bang on, just commented in another thread to say to change color/size/style you can pass that down if you wanted to. I don't think we need it here at the moment.